### PR TITLE
Password rule improvement

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Fix false positive "passing" and "passes" cases for Password rule

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Test Benchmark (-4 false-positives)

### CODE REVIEW

File: credsweeper/credentials/line_data.py [hunk 1]
```diff
@@ -112,7 +112,9 @@ def clean_bash_parameters(self) -> None:
 
     def sanitize_variable(self) -> None:
         """Remove trailing spaces, dashes and quotations around the variable."""
-        if self.variable:
+        mood_var_len = 0
```
Comment (csh519) [inline credsweeper/credentials/line_data.py:115]: May I ask why you named variable name like `mood_var_len`?

File: credsweeper/credentials/line_data.py [hunk 2]
```diff
@@ -112,7 +112,9 @@ def clean_bash_parameters(self) -> None:
...
+        while self.variable and mood_var_len != len(self.variable):
+            mood_var_len = len(self.variable)
```
Comment (babenek) [inline credsweeper/credentials/line_data.py:117]:
```suggestion
        mud_var_len = 0
        while self.variable and mud_var_len != len(self.variable):
            mud_var_len = len(self.variable)
```

File: credsweeper/credentials/line_data.py (hunk 1 above)
Comment (babenek) [reply credsweeper/credentials/line_data.py]: My fail in grammatik :)

**Review by csh519 — APPROVED** [review]
Thank you for improve the performance of Password rule.

LGTM :+1:

**Review by iuriimet — APPROVED** [review]

**Pull request merged**